aws/resource_aws_chime_voice_connector_termination_test.go

```diff
@@ -0,0 +1,160 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/chime"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccAWSChimeVoiceConnectorTermination_basic(t *testing.T) {
+	var vc *chime.VoiceConnector
+	name := acctest.RandomWithPrefix("tf-acc-test")
+	vcResourceName := "aws_chime_voice_connector.chime"
+	resourceName := "aws_chime_voice_connector_termination.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		ErrorCheck:   testAccErrorCheck(t, chime.EndpointsID),
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAWSChimeVoiceConnectorDestroy,
```

Comment:
since voice connector termination is its own resource, we can verify destroy functionality as a separate method e.g.

```go
func testAccCheckAWSChimeVoiceConnectorTerminationDestroy(s *terraform.State) error {
	for _, rs := range s.RootModule().Resources {
		if rs.Type != "aws_chime_voice_connector_termination" {
			continue
		}
		conn := testAccProvider.Meta().(*AWSClient).chimeconn
		input := &chime.GetVoiceConnectorTerminationInput{
			VoiceConnectorId: aws.String(rs.Primary.ID),
		}
		resp, err := conn.GetVoiceConnectorTerminationWithContext(context.TODO(), input)

		if isAWSErr(err, chime.ErrCodeNotFoundException, "") {
			continue
		}

		if err != nil {
			return err
		}

		if resp != nil && resp.Termination != nil {
			return fmt.Errorf("error Chime Voice Connector Termination still exists")
		}
	}

	return nil
}
```

Reply:
I will add this one. Good call!